src/speakbuttons.cpp

```diff
@@ -123,6 +123,27 @@ void SpeakButtons::setEmotion(QOnlineTranslator::Engine engine, QOnlineTts::Emot
     }
 }
 
+QMap<QOnlineTranslator::Language, QLocale::Country> SpeakButtons::regions(QOnlineTranslator::Engine engine) const
+{
+    switch (engine) {
+    case QOnlineTranslator::Google:
+        return m_googleRegions;
+    default:
+        return {};
+    }
+}
+
+void SpeakButtons::setRegions(QOnlineTranslator::Engine engine, const QMap<QOnlineTranslator::Language, QLocale::Country> &regions)
```

Comment:
`std::move` won't have any effect if you use it on a const reference. For more details see [this](https://clang.llvm.org/extra/clang-tidy/checks/modernize/pass-by-value.html).
```suggestion
void SpeakButtons::setRegions(QOnlineTranslator::Engine engine, QMap<QOnlineTranslator::Language, QLocale::Country> regions)
```

Reply:
Oh, I don't know about that!
I only once read that in modern C++ one should move instead of copy large objects and didn't know many details.

Reply:
Yeah, it's tricky. CI will catch it, but I assumed that you may be interested in the explanation.